src/syzygy/syzygy.cc

```diff
@@ -1066,15 +1065,12 @@ class SyzygyTablebaseImpl {
                                     size_low, nullptr);
     CloseHandle(fd);
     if (!mmap) {
-      CERR << "CreateFileMapping() failed";
-      exit(1);
+      throw Exception("CreateFileMapping() failed");
     }
     *mapping = mmap;
     base_address = MapViewOfFile(mmap, FILE_MAP_READ, 0, 0, 0);
     if (!base_address) {
-      CERR << "MapViewOfFile() failed, name = " << fname
-           << ", error = " << GetLastError();
-      exit(1);
+      throw Exception("MapViewOfFile() failed, name = " + fname + ", error = " + GetLastError());
```

Comment:
This doesn't compile for windows, as it's not possible to add number to the string.
It should be `std::to_string(GetLastError())`.

Reply:
@mooskagh Does 213cf4d fix it?